.github/actions/send-telegram-notify/action.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Send Telegram message on failure'
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        python -m pip install --upgrade pip
+        pip install requests
+      shell: bash
+    # Let's logout github environment
+    - env:
+        GITHUB_CONTEXT: ${{ toJson(github) }}
+      run: echo "$GITHUB_CONTEXT"
+      shell: bash
+    - env:
+        MSG: >-
+          🔴 Workflow testing failed:\n
+          Job: [ `${{ github.job }}`](https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }})\n
+          Commit: [ `${{ github.sha }}`](https://github.com/${{ github.repository }}/commit/${{ github.sha }})\n
+          Branch: [ `${{ env.CI_ACTION_REF_NAME }}`](https://github.com/${{ github.repository }}/tree/${{ github.ref }})\n
```

Comment:
NB: A whitespace is part of a link here.

Reply:
It was noticed that it failed to show the link w/o this space using previous external Action. It could be some in issue from that Action, but may be it's better to have it to avoid of possible failures.

Reply:
I would try again. And I guess we can remove backticks inside a link text, because, as I see, they don't change a font that is used for links.

Reply:
I've just retired it w/o spaces and got unworkable links in the message
🔴 Workflow testing failed:
 Job: release
 Commit: d998e6b409d6f669e682a00898855b8fa31d3ff1
 Branch: avtikhon/github-common
 History: commits (https://github.com/tarantool/tarantool/commits/d998e6b409d6f669e682a00898855b8fa31d3ff1)
 Triggered on: push
 Committer: avtikhon 
 -------------------- Commit message --------------------------
 remove spaces

Reply:
Remove of backticks broke message appearance in Telegram bot at all.